### PR TITLE
use config file

### PR DESCRIPTION
**Pull request by zarsky-broad**

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -4,6 +4,7 @@ import { a, div, h, h1, h2, nav } from 'react-hyperscript-helpers'
 import * as Nav from '../nav'
 import * as Style from '../style'
 import * as Utils from '../utils'
+import * as Config from './config'
```
Comment (dmohs) [inline src/components/App.js:7]: Suggest alphabetize so we don't have to think about order.

File: src/components/config.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+let loadedConfig
+
+const loadConfig = () =>
+  fetch('config.json').then(response => response.json()).then(json => loadedConfig = json)
+
+
+const getRawlsUrlRoot = () => loadedConfig['rawlsUrlRoot']
+const getGoogleClientId = () => loadedConfig['googleClientId']
```
Comment (dmohs) [inline src/components/config.js:8]: Same suggestion about alphabetizing.

**Review by dmohs — APPROVED** [review]

File: src/components/App.js (hunk 1 above)
Comment (zarsky-broad) [reply src/components/App.js:7]: So, this is the result of the alphabetizing by IntelliJ, which I believe is by path. I'm taking a stab at updating in my next PR.

File: src/components/config.js (hunk 2 above)
Comment (zarsky-broad) [reply src/components/config.js:8]: This is the order they appear in inside of the config.json itself.

File: src/components/App.js (hunk 1 above)
Comment (dmohs) [reply src/components/App.js:7]: That's fine by me. I don't care what the convention is, just that I don't have to spend brain cycles on where to put things.

File: src/components/config.js (hunk 2 above)
Comment (dmohs) [reply src/components/config.js:8]: Okay, let's merge this and reorder those in a follow-up.

**Pull request merged**